src/components/ErrorPage.tsx

```diff
@@ -0,0 +1,21 @@
+import _ from 'lodash';
+import React from 'react';
+
+interface Props {
+  error: string;
+}
+
+const initialState = { error: "Application Error" };
+type State = Readonly<typeof initialState>;
+
+export default class ErrorPage extends React.PureComponent<Props, State> {
+  readonly state: State = initialState;
+
+  constructor(props) {
+    super(props);
+  }
+
+  render() {
+    return <div> Error: {this.state.error} </div>;
```

Comment:
I didn't quite get around to displaying the error message that was set by the reducer -- how do you do this again?

Reply:
The `error` is stored in the redux store and you've made that available by binding the store to the component in the `ErrorContainer`. All you need to do now is read the error from the props i.e. use `this.props.error`. You can remove the error from the state (and actually remove references to the state completely).